documentation/feature/medical-system.md

```diff
@@ -288,5 +296,76 @@ Note that keeping the patient's vitals stable is very important while treating h
 - You don't have to take epinephrine after you take morphine, just wait until your pulse stabilizes by itself (Provided that you are in a stable condition).
 - Giving too much morphine to a patient (more than one every 10 minutes) will put him in cardiac arrest because of a blood pressure / heart rate too low.
 
-## 3. Dependencies
-`ace_interaction`, `ace_modules`, `ace_apl`
\ No newline at end of file
+#### 2.2.5 Revive (Advanced Medical)
+For the following procedure to work revive need to be enabled.
+
+- A unit in the revive state will be unconscious and will stay unconscious until he is either woken up or the revive timer runs out.
+- A unit in the revive state can't die from any source of damage, only the timer ending can kill it.
+- To wake up a patient the use of a PAK is required.
+- Each successful revive removes a life from the unit, once the lives run out the next time unit will take fatal damage he will not enter the revive state and will die.
+- Each successful round of CPR (filled up completion bar) increase the time left in the revive state.
+
+## 3. Guides
+### 3.1 Loadouts
+
+#### 3.1.1 Basic
+- Soldier:
+  - 10 × Bandage (Basic)
+  - 3 × Morphine autoinjector
+  - 1 × Epinephrine autoinjector
+
+- Medic:
+  - 15-25 × Bandage (Basic)
+  - 10 × Morphine autoinjector
+  - 10 × Epinephrine autoinjector
+  - 6 × Blood IV (500ml)
+
+#### 3.1.2 Advanced
+
+- Soldier :
+  - 3-6 × Bandage (Basic)
+  - 3-6 × Bandage (Elastic)
+  - 3-6 × Packing Bandage
+  - 3-6 × Basic Field Dressing (QuikClot)
+  - 1 × Morphine autoinjector
+  - 1 × Epinephrine autoinjector
+  - 1 × Tourniquet (CAT)
+  - **Optional**: 1 × Saline IV (500ml) - used only by qualified medic
+
+
+- Combat First Responder (CFR):
+  - 10-15 × Bandage (Basic)
+  - 10-15 × Bandage (Elastic)
+  - 10-15 × Packing Bandage
+  - 10-15 × Basic Field Dressing (QuikClot)
+  - 8 × Atropine autoinjector
+  - 5 × Morphine autoinjector
+  - 5 × Epinephrine autoinjector
+  - 3 × Tourniquet (CAT)
+  - 4 × Saline IV (500ml)
+
+
+- Medic:
+  - 10-15 × Bandage (Basic)
+  - 15-20 × Bandage (Elastic)
+  - 15-20 × Packing Bandage
+  - 10-15 × Basic Field Dressing (QuikClot)
+  - 12 × Atropine Autoinjector
```

Comment:
I went for lower-case because it's named like that in the actual game/stringtable.